tagesspiegel.de.txt

```diff
@@ -1,60 +1,34 @@
-# Author: zinnober
-# Should work with "normal" articles as well as with image galleries
-
-prune: no
-
 # Title
-title: //h1/span[@class='hcf-headline']
+title: //div[@class='ts-title']
 
 # Set author
 author: //a[@rel='author']
 
 # Set date
-date: //span[@class='date hcf-atlas']
+date: //time[@class='ts-time']
 
 # Fetch full multipage articles
-next_page_link: //a[contains(@class, 'hcf-forward')]
+single_page_link: //a[contains(@class, 'ts-one-page')]
 
 # Content is here
-body: //article
-body: //div[contains(@class, 'hcf-screen')]
-
-# Remove tracking and ads
-strip_id_or_class: hcf-ad
-strip_id_or_class: hcf-autoload-ad
-strip_id_or_class: hcf-content-ad
-
-# Tidy up before article
-strip: //article/h1
-strip_id_or_class: hcf-atlas
-strip_id_or_class: hcf-author
-strip_id_or_class: date hcf-atlas
-strip_id_or_class: date hcf-atlas
+body: //article[@class='ts-article']
 
 # General cleanup
-strip: //div[contains(@class, 'hcf-screen')]//h1
-strip: //div[@class='hcf-subpage-titles']//ul
-strip_id_or_class: hcf-doctype-media
-strip_id_or_class: hcf-inline-gallery
-strip_id_or_class: hcf-doctype-video
-strip_id_or_class: hcf-links
-strip_id_or_class: hcf-mini-navi
-strip_id_or_class: hcf-media-control
+strip: //iframe
 strip_id_or_class: hcf-hidden
-replace_string(<span class="hcf-update">Update</span>): <strong>Update: </strong>
+strip_id_or_class: ts-user-quote
+strip_id_or_class: ts-abo-link
+strip_id_or_class: ts-homepage-link
+strip_id_or_class: ts-recommendation
 
 # Fix pictures and captions
-replace_string(<a class="hcf-doctype-gallery): <p class="hcf-doctype-gallery
-replace_string(<a class="hcf-doctype-enlarge): <p class="hcf-doctype-enlarge
-replace_string(<figcaption class="hcf-caption">): <br><small><em>
-replace_string(</figcaption>): </em></small>
-
-# Fix image galleries
+replace_string(<a class="ts-gallery): <p class="ts-gallery
+replace_string(<figcaption class="ts-caption">): <br><small><em>                                                                                                                                                                             replace_string(</figcaption>): </em></small>                                                                                                                                                                                                                                                                                                                                                                                                                                              # Fix image galleries
```

Comment:
Be careful, that should be on multiple lines instead of one

```suggestion
replace_string(<figcaption class="ts-caption">): <br><small><em>
replace_string(</figcaption>): </em></small>
# Fix image galleries
```

Reply:
Whoops. Thanks.